contributingGuide.md

```diff
@@ -0,0 +1,64 @@
+# Contributing Guide
```

Comment:
There is already an existing contribution guide under CONTRIBUTE.md. It was mentioned in the ticket that @harshala334 will take that over. Feel free to add any additions to that one.